calyx/src/passes/par_to_seq.rs

```diff
@@ -0,0 +1,45 @@
+use crate::ir::traversal::{Action, Named, VisResult, Visitor};
+use crate::ir::{self, LibrarySignatures};
+
+#[derive(Default)]
+/// Transforms all `par` into `seq`.
+/// Useful for debugging problems with `par`.
+///
+/// # Example
+/// 1. Collapses nested `seq`:
```

Comment:
Silly question, maybe, but perhaps "collapses" is not the right word here? In this example, nothing gets collapsed—it's a straight replacement.

Reply:
Ah, this is a copy-pasta